### DISCUSSION TITLE
Vulnerabilities update after OS upgrade

### DISCUSSION THREAD

**Discussion by eglyn** [wazuh/wazuh#14398] [General]
Hi, 

Simple question: how to remove old vulnerabilities when updating the OS?
For example, I updated a Debian 10 to a Debian 11, it's been 2 days, and in the vulnerabilities, I still have those of Debian 10...

thx !

**Comment by eglyn** [comment]
It seems there is no vulnerability scan since OS has been upgraded to Debian 11 :/

**Comment by mdiego92** [comment]
Hello eglyn,

The vulnerability detector currently is supported for Debian 11, as you can see in the following documentation for **Wazuh 4.3**:
https://documentation.wazuh.com/current/user-manual/capabilities/vulnerability-detection/compatibility-matrix.html

Could you please confirm what version of Wazuh are you using?

As per our documentation, the **Wazuh 4.2** and **previous versions** do not support Debian 11. If that's the case, you could check our documentation article to [scan vulnerabilities on unsupported systems](https://documentation.wazuh.com/4.2/user-manual/capabilities/vulnerability-detection/allow-os.html).

Please let us know if you have any remaining questions.

Thank you very much.
